docs/source/workflows/design_workflows.rst

```diff
@@ -212,3 +212,33 @@ Thus, archiving all the design workflows contained within a branch will result i
 
     # Restore the branch to active status.
     my_branch = project.branches.restore(my_branch.uid)  # my_branch.archived == False
+
+You can also update the data on a branch in a similar way that one updates to later date via the web application.
+To do so, you can first grab the available updates to a branch:
```

Comment:
No need to repeat "you can" so much. Just say "First, grab the available updates to a branch:"